src/Features/Lsif/GeneratorTest/SemanticTokensTests.vb

```diff
@@ -0,0 +1,84 @@
+                Dim tokens = lsif.GetSemanticTokens(document)
+                Dim serializedTokens = JsonConvert.SerializeObject(tokens)
+
+                Assert.Equal(expectedTokens, serializedTokens)
+            End Using
+        End Function
+
+        <Fact>
+        Public Async Function TestSemanticTokensCapability() As Task
+            ' This test validates that the expected LSIF token types are generated.
+            '
+            ' Roslyn's notion of 'semantic tokens' doesn't follow the strict definition
+            ' of syntax vs semantic, it's divided up into in-proc quick-tokenization
+            ' colorizer and _everything else_, generated via LSP.
+            '
+            ' Because of this we emit both semantic and syntatic tokens so that consumers
+            ' of the LSIF format have a single consistent palette of colors without missing
+            ' some traditionally 'semantic' tokens.
+            Const SerializedSemanticTokensCapabilities = "{""tokenTypes"":[""namespace"",""type"",""class"",""enum"",""interface"",""struct"",""typeParameter"",""parameter"",""variable"",""property"",""enumMember"",""event"",""function"",""method"",""macro"",""keyword"",""modifier"",""comment"",""string"",""number"",""regexp"",""operator"",""class name"",""constant name"",""delegate name"",""enum member name"",""enum name"",""event name"",""excluded code"",""extension method name"",""field name"",""interface name"",""json - array"",""json - comment"",""json - constructor name"",""json - keyword"",""json - number"",""json - object"",""json - operator"",""json - property name"",""json - punctuation"",""json - string"",""json - text"",""keyword - control"",""label name"",""local name"",""method name"",""module name"",""namespace name"",""operator - overloaded"",""parameter name"",""preprocessor keyword"",""preprocessor text"",""property name"",""punctuation"",""record class name"",""record struct name"",""regex - alternation"",""regex - anchor"",""regex - character class"",""regex - comment"",""regex - grouping"",""regex - other escape"",""regex - quantifier"",""regex - self escaped character"",""regex - text"",""string - escape character"",""string - verbatim"",""struct name"",""text"",""type parameter name"",""whitespace"",""xml doc comment - attribute name"",""xml doc comment - attribute quotes"",""xml doc comment - attribute value"",""xml doc comment - cdata section"",""xml doc comment - comment"",""xml doc comment - delimiter"",""xml doc comment - entity reference"",""xml doc comment - name"",""xml doc comment - processing instruction"",""xml doc comment - text"",""xml literal - attribute name"",""xml literal - attribute quotes"",""xml literal - attribute value"",""xml literal - cdata section"",""xml literal - comment"",""xml literal - delimiter"",""xml literal - embedded expression"",""xml literal - entity reference"",""xml literal - name"",""xml literal - processing instruction"",""xml literal - text""],""tokenModifiers"":[]}"
```

Comment:
Ditto as earlier -- let's find a way to avoid hard-coding another copy of this list.

Reply:
I'd also be OK deleting this in favor of the existing output format tests, since I don't know if this adds much.